app/controllers/PensionDetailsController.scala

```diff
@@ -68,12 +68,12 @@ class PensionDetailsController @Inject()(authAction: AuthAction,
           },
           pensionDetails => {
 
-            val validateSconRequest = ValidateSconRequest(pensionDetails.scon.toUpperCase)
+            val validateSconRequest = ValidateSconRequest(pensionDetails.scon)
```

Comment:
I think the controller had tests for these requests. However, they only catered for caps S.
But as we expect the controller to work for small case aswell - lets add the small case prefix test scenario to the controller test.